daemon/logger/loginfo.go

```diff
@@ -7,8 +7,8 @@ import (
 	"time"
 )
 
-// Context provides enough information for a logging driver to do its function.
-type Context struct {
+// LogInfo provides enough information for a logging driver to do its function.
```

Comment:
As we already are in `logger` package, `LogInfo` seems redondant, maybe just `Info` ? @thaJeztah wdyt 👼 ?